plugins/woocommerce/includes/admin/class-wc-admin.php

```diff
@@ -267,44 +266,6 @@ public function preview_emails() {
 		}
 	}
 
-	/**
-	 * Preview email editor placeholder dummy content.
-	 */
-	public function preview_email_editor_dummy_content() {
```

Comment:
Theoretically, this was public in the woocommerce/includes folder, so removing it is a BC break. Personally I, think that there is very little chance that someone used this, so I think we can take the risk and remove it.

Reply:
@costasovo, do you think it's worth changing the significance in the changelog to `major` since it's breaking backwards compatibility?

Reply:
I'm not sure what the effect would be. AFAIK, Woo plugin versioning is not affected by that. So I have no idea, to be honest.